doc/source/merge_changelogs.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Append release notes from other projects of the larray-project constellation to the current changelog
+"""
+
+import os
+import requests
+from subprocess import check_call
+
+
+# TODO: add Eurostat project
+EDITOR_GITHUB_REP = "https://github.com/larray-project/larray-editor"
+
+
+def short(rel_name):
+    return rel_name[:-2] if rel_name.endswith('.0') else rel_name
+
+
+def relname2fname(release_name):
+    short_version = short(release_name)
+    return r"version_{}.rst.inc".format(short_version.replace('.', '_'))
+
+
+def yes(msg, default='y'):
+    choices = ' ({}/{}) '.format(*tuple(c.capitalize() if c == default else c
+                                        for c in ('y', 'n')))
+    answer = None
+    while answer not in ('', 'y', 'n'):
+        if answer is not None:
+            print("answer should be 'y', 'n', or <return>")
+        answer = input(msg + choices).lower()
+    return (default if answer == '' else answer) == 'y'
+
+
+def include_external_changelogs(section_name, release_name, external_github_rep, rel_changes_dir='/doc/source/changes',
+                                branch='master'):
+    changelog_file = relname2fname(release_name)
+    short_version = short(release_name)
+
+    # get external changelogs
+    url = external_github_rep.replace('github.com', 'raw.githubusercontent.com') \
+          + '/{}/{}/'.format(branch, rel_changes_dir) + changelog_file
+    req = requests.get(url)
+    if req.status_code != requests.codes.ok:
+        raise ValueError("Content at URL {} cound not been found.".format(url))
+    external_changelog_content = req.text
+
+    # dump external changelogs
+    changelog_file = os.path.abspath(os.path.join('.', 'changes', changelog_file))
+    with open(changelog_file) as f:
+        lines = f.readlines()
```

Comment:
Is there a reason why you use .readlines(), splitlines() and writelines() instead of just using .read() and .write()? If so, please add a comment.